validation.md

```diff
@@ -2327,13 +2327,20 @@ Each method on the `Email` rule addresses a different aspect of email validation
 
 #### `rfcCompliant(strict: bool)`
 
-**What It Does**
+**What it does**
 - Validates the email according to [RFC 5322](https://datatracker.ietf.org/doc/html/rfc5322).
-- Passing `true` enforces stricter RFC checks (e.g., rejects `invalid.@example.com`).
+- Passing `true` enforces stricter RFC checks (e.g., rejects trailing dots or multiple consecutive dots).
 
-**When to Use It**
-- Use `rfcCompliant()` if you need standard RFC validation while allowing some unusual formats.
-- Use `rfcCompliant(true)` if you want to reject less typical but technically valid addresses.
+**When to use it**
+- Use `rfcCompliant()` for typical RFC validation while allowing some uncommon formats.
+- Use `rfcCompliant(true)` to reject unusual but technically valid addresses.
+
+**Examples**
+- **Addresses that pass on `rfcCompliant()` but fail on `rfcCompliant(true)`**
+  - `invalid.@example.com` (local part ends with a dot)
+  - `some..dots@example.com` (double dots in local part)
+- **Address that fail on both**
+  - `test@example..com` (double dots in the domain part) typically fails in *both* modes.
 
 **Example**
```

Comment:
Since we now have both "examples" and "example", it'd suggest renaming the latter:
```suggestion
**Examples**
- **Addresses that pass on `rfcCompliant()` but fail on `rfcCompliant(true)`**
  - `invalid.@example.com` (local part ends with a dot)
  - `some..dots@example.com` (double dots in local part)
- **Address that fail on both**
  - `test@example..com` (double dots in the domain part) typically fails in *both* modes.

**Code sample**
```